homeassistant/components/control4/director_utils.py

```diff
@@ -38,6 +40,32 @@ async def director_update_data(
     return {key["id"]: key for key in data}
 
 
+async def update_variables_for_entity(
```

Comment:
Is there a reason that we couldn't use this function for the light component as well? This seems like it could possibly replace `director_update_data`, which would be preferable to avoid duplication

Reply:
Yes, I believe so. For my current work I just wanted to focus on the media player and make no changes to other portions of the integration in case of breakage.

Reply:
Done, I tested it on my own lights which include dimmable and non-dimmable.